src/main/java/org/folio/search/service/IndexService.java

```diff
@@ -47,15 +48,18 @@ public class IndexService {
    * @throws SearchServiceException if {@link IOException} has been occurred during index request execution
    */
   public FolioCreateIndexResponse createIndex(String resourceName, String tenantId) {
+    log.debug("createIndex:: by [resourceName: {}, tenantId: {}]",
+      resourceName, tenantId);
+
     validateResourceName(resourceName,
       "Index cannot be created for the resource because resource description is not found.");
 
     var index = getIndexName(resourceName, tenantId);
     var settings = settingsHelper.getSettings(resourceName);
     var mappings = mappingHelper.getMappings(resourceName);
 
-    log.info("Creating index for resource [resource: {}, tenant: {}, mappings: {}, settings: {}]",
-      resourceName, tenantId, mappings, settings);
+    log.info("Attempts to create index by [indexName: {}, mappings: {}, settings: {}]",
+      index, mappings, settings);
```

Comment:
We have multitenancy in system so It's needed to add tenantId, where it's available